### ISSUE TITLE
PHP8 hansott/psr7-cookies

### ISSUE THREAD

**Issue by pavarnos** [WyriHaximus/reactphp-http-middleware-session#41]
Hi
Do you have plans for a php 8 compatible release? hansott/psr7-cookies v2.0.2 only allows php 7... v3 allows php 8

```
> composer prohibits php:8
hansott/psr7-cookies                       2.0.2  requires  php (~7.0)  
wyrihaximus/react-http-middleware-session  4.0.0  requires  php (^7.3)  
```

I see #37 has a lot of WIP, so I'm not sure if making a small pull request would help you here?

**Comment by WyriHaximus** [comment]
> Hi
> Do you have plans for a php 8 compatible release? hansott/psr7-cookies v2.0.2 only allows php 7... v3 allows php 8
> 
> ```
> > composer prohibits php:8
> hansott/psr7-cookies                       2.0.2  requires  php (~7.0)  
> wyrihaximus/react-http-middleware-session  4.0.0  requires  php (^7.3)  
> ```

In short, I will be updating all my packages to support PHP 8 in time, but for now, it's just on demand by issues like this one when someone raises it. As you've seen in #37 I've been working on aligning all my packages, and once that's done automated aligning should be in place to make that easier.

> I see #37 has a lot of WIP, so I'm not sure if making a small pull request would help you here?

Yeah I need to get back to that, but I'm surely open to a PR adding PHP 8 support. IIRC while doing #37 in `hansott/psr7-cookies` v3 they removed something this package relies on.

**Comment by WyriHaximus** [comment]
@pavarnos Just completed and merged #37

**Comment by pavarnos** [comment]
see #42
